docs/getting-started.md

```diff
@@ -253,7 +253,7 @@ If you would like to run specific functional tests only, you can leverage `ginkg
 command line options as follows (run a specified suite):
 
 ```
-    FUNC_TEST_ARGS='-focus-file=vmi_networking_test' make functest
+    FUNC_TEST_ARGS='-focus-file=vmi_networking' make functest
```

Comment:
nit: Please consider adding a link to Ginkgo's [Location-Based Filtering](https://onsi.github.io/ginkgo/#location-based-filtering) as a note for people who wishing to understand how this flag works.

Reply:
Good idea. Done